types/git-clone/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "DOM",
```

Comment:
```suggestion
```
is it required by this lib?
just rewrite your `console` to type assertions:
https://github.com/DefinitelyTyped/DefinitelyTyped#my-package-teststs